projects/igniteui-angular/src/lib/avatar/avatar.component.ts

```diff
@@ -227,6 +213,7 @@ export class IgxAvatarComponent implements OnInit, AfterViewInit {
     @Input()
     public src: string;
 
+    private _size: string | Size = 'small';
```

Comment:
Maybe we can collapse the union just to `Size` type

Reply:
It's done